mbofui/app/js/controllers.js

```diff
@@ -353,9 +353,11 @@ ocellus.controller('mapController', ['$compile', '$scope', '$rootScope','$filter
   };
 
   $scope.eventEditCancel = function(){
-    $scope.editEvent.status = 'canceled';
-    $scope.removeEvent($scope.editEvent);
-    $scope.eventEditPost();
+    if (window.confirm("Are you sure you want to cancel this event?")) {
+      $scope.editEvent.status = 'canceled';
+      $scope.removeEvent($scope.editEvent);
+      $scope.eventEditPost();
+    }
   };
```

Comment:
Although I think this is a good idea, I can see it as being another step for users to simply cancel an event. Is there a way users could turn this feature off? Maybe something for 2.0?